lib/src/fnc/mod.rs

```diff
@@ -289,8 +289,16 @@ pub fn synchronous(ctx: &Context<'_>, name: &str, args: Vec<Value>) -> Result<Va
 		"type::table" => r#type::table,
 		"type::thing" => r#type::thing,
 		//
+		"vector::add" => vector::add,
+		"vector::angle" => vector::angle,
+		"vector::crossproduct" => vector::crossproduct,
 		"vector::dotproduct" => vector::dotproduct,
```

Comment:
This issue existed prior to this PR, but my preference is to name these `vector::dot` and `vector::cross` for conciseness or, if we must, `vector::product::dot` and `vector::product::cross`, for consistency.

Reply:
Agreed, let's be concise